emr/application/controllers/Request.php

```diff
@@ -11,42 +11,133 @@ public function __construct() {
-       $data["font"] = ["database","user-plus", "edit", "refresh", "sign-out"];
-
-       $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
-       $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
-       $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];    
-     } elseif ($this->ion_auth->in_group("hostpial admin")) {
-       $data["userRole"] = "HOSPITAL ADMIN";
-       $data["options"] = ["Logout"];
-       $data["href"] = ["auth/logout"];
-       $data["font"] = ["refresh", "sign-out"];
-
-       $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
-       $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
-       $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];   
-    } else {
-      $data["userRole"] = "SCREENER";
-      $data["options"] = ["Logout"];
-      $data["href"] = ["auth/logout"];
-      $data["font"] = ["sign-out"];
-
-      $data["sideMenu"] = ["Calendar", "Availability"];
-      $data["link"] = ["main/index", "screeners/add"];
-      $data["icon"] = ["calendar","user"];
+        $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
+        $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
+        $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];    
+      } elseif ($this->ion_auth->in_group("hostpial admin")) {
+        $data["userRole"] = "HOSPITAL ADMIN";
+        $data["options"] = ["Logout"];
+        $data["href"] = ["auth/logout"];
+        $data["font"] = ["refresh", "sign-out"];
+
+        $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
+        $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
+        $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];   
+      } else {
+        $data["userRole"] = "SCREENER";
+        $data["options"] = ["Logout"];
+        $data["href"] = ["auth/logout"];
+        $data["font"] = ["sign-out"];
+
+        $data["sideMenu"] = ["Calendar", "Availability", "My Requests"];
+        $data["link"] = ["main/index", "screeners/add", "request/view"];
+        $data["icon"] = ["calendar","user", "check-square-o"];
+      }
+
+      $data["userFirstName"] = $this->ion_auth->user()->row()->first_name;
+      $data["userLastName"] = $this->ion_auth->user()->row()->last_name;
+
+      $this->load->view('main/header');
+      $this->load->view('main/sidebar', $data);
+      $this->load->view('main/topbar', $data);
+      $this->load->view('screeners/request');
+      $this->load->view('main/footer');
+    }
+
+    public function addRequest() {
```

Comment:
should be renamed to just `add` instead of `addRequest`

Reply:
done

Reply:
Looks good!